lib/pacemaker/pcmk_sched_bundle.c

```diff
@@ -816,13 +822,11 @@ add_replica_actions_to_graph(pcmk__bundle_replica_t *replica, void *user_data)
         }
     }
     if (replica->ip != NULL) {
-        replica->ip->cmds->add_actions_to_graph(replica->ip);
-    }
-    if (replica->container != NULL) {
-        replica->container->cmds->add_actions_to_graph(replica->container);
+        replica->ip->private->cmds->add_actions_to_graph(replica->ip);
     }
+    replica->container->private->cmds->add_actions_to_graph(replica->container);
```

Comment:
There's a Coverity error here because *almost* everywhere we assume `replica->container` is non-NULL, but we NULL-check it in the `if` condition at the top of this function.
```
14:14:41  ===== NEW COVERITY SCAN ERRORS DETECTED =====
14:14:41  Error: FORWARD_NULL (CWE-476):
14:14:41  /pacemaker/lib/pacemaker/pcmk_sched_bundle.c:781: path: Condition "replica->remote != NULL", taking true branch.
14:14:41  /pacemaker/lib/pacemaker/pcmk_sched_bundle.c:781: path: Condition "replica->container != NULL", taking false branch.
14:14:41  /pacemaker/lib/pacemaker/pcmk_sched_bundle.c:781: var_compare_op: Comparing "replica->container" to null implies that "replica->container" might be null.
14:14:41  /pacemaker/lib/pacemaker/pcmk_sched_bundle.c:824: path: Condition "replica->ip != NULL", taking true branch.
14:14:41  /pacemaker/lib/pacemaker/pcmk_sched_bundle.c:827: var_deref_op: Dereferencing null pointer "replica->container".
14:14:41  ===== END NEW COVERITY SCAN ERRORS DETECTED =====
```

Reply:
Yep every replica has to have a container ... we could drop the check there, but we do need to check it in the free function, which can be called by the create function on failures before the container is added

Reply:
That check is in the same function as the use it's complaining about, so hopefully just dropping that check will be sufficient. Updated commit.